Scripts/zsh_plugin_installer.py

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/python3
+
+# |---/ /+----------------------------------------+---/ /|#
+# |--/ /-| Script to install pkgs from input list |--/ /-|#
+# |-/ /--| Effibot                                |-/ /--|#
+# |/ /---+----------------------------------------+/ /---|#
+
+import argparse
+
+# get reference to directories
+# ZSH_FOLDER="/usr/share/oh-my-zsh"
+import os
+from os.path import exists, expanduser, isdir, join
+
+# define folders
+global ZSH_FOLDER
+global ZSH_CUSTOM_PLUGINS
+global WORKARAOUND
+global RC_FILE
+# those folders are hardcoded for now because the script is
+# meant to be used with the configuration of main install script
+ZSH_FOLDER = expanduser("/usr/share/oh-my-zsh")
+ZSH_CUSTOM_PLUGINS = join(ZSH_FOLDER, "custom/plugins")
+WORKARAOUND = False
+RC_FILE = expanduser("~/.zshrc")
+# this is a workaround string to be able to load the completion plugin manually
```

Comment:
this `RC_FILE` is pointing to the root user as this script is executed as `sudo` --> `/root/.zshrc`